Tests/test_imagedraw.py

```diff
@@ -436,18 +437,22 @@ def test_square(self):
         expected.load()
         img, draw = self.create_base_image_draw((10, 10))
         draw.polygon([(2, 2), (2, 7), (7, 7), (7, 2)], BLACK)
+        del draw
```

Comment:
Why is it important?

Reply:
It's not. I just thought it would be nice to clean up after the test code. I can remove the commit if you like.

Reply:
They're just going to go out of scope at the end of the method, like the image.

Reply:
Okay, I've removed the commit.